src/BlazorUI/Bit.BlazorUI/Components/Inputs/Dropdown/BitDropdown.scss

```
@@ -749,24 +749,24 @@
 .bit-drp-cal {
     display: none;
     position: fixed;
-    font-weight: 400;
+    font-weight: $tg-fw-regular;
     overflow: hidden;
     box-sizing: border-box;
     z-index: $zindex-callout;
     font-size: var(--bit-drp-fs);
     animation-fill-mode: both;
     @include drp-size-medium;
     animation-duration: $mot-duration-long;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)